content/communityExtensions/figmaToBabylonJS.md

```diff
@@ -0,0 +1,69 @@
+keywords: extensions, Figma, GUI, BabylonGUI, Design, UI, plugin
+further-reading:
+video-overview:
+video-content:
+---
+
+# Figma to BabylonJS
+
+## Introduction
+
+![Screenshot showing the figma right click menu on the Plugins option](/img/extensions/FigmaToBabylonJS.png)
+
+FigmaToBabylonJS is a Figma plugin which allows you to export Figma designs to BabylonJS GUI .json files. These can be parsed into a fullscreen UI [Advanced Dynamic Texture](/features/featuresDeepDive/gui/gui#advanceddynamictexture).
+
+
+## Get the Plugin
+
+The plugin is available via the [Figma community](https://www.figma.com/community/plugin/1186201881571137432) page.
+
+## How to Use
+
+After adding the plugin to Figma, you need to highlight the frame/artboard that you want to export and right click on it.
+On the rightclick menu, select Plugins > Saved Plugins > FigmaToBabylonJS
+![Screenshot showing the figma right click menu on the Plugins option](/img/extensions/FigmaToBabylonJS-Step1.png)
+
+Choose if you want the output to include the background color (changes the "root" background color to the frames fill color)
+Click the “Convert to BabylonJS JSON” button and after a short wait you will see your file explorer. Choose where you want the file to go. For best results, place it directly into your project with the correct file name your code uses, this way the UI will automatically update.
+![Screenshot showing the figma right click menu on the Plugins option](/img/extensions/FigmaToBabylonJS-Step2.png)
+
+It's also good practice to load the UI in gui.babylonjs.com or in your project but loading in the inspector so you can edit the UI after the fact. Just to make sure that the exporting has went the way you expected and there weren't any strange inconsistencies with the design [see inconsistencies and polyfills](/communityExtensions/figmaToBabylonJS#inconstancies--polyfills)
+
+## How it Works
+
+The plugin recursively loops through the selected Figma frame and pulls out all the "nodes" in the design. It will take each node and find a correct BabylonJS GUI object to map to it.
+All the new BabylonJS GUI mapped objects are then added to a empty root container object as children. When the plugin is finished adding all these children, it simply outputs it as a JSON string and writes it to a file and makes it available to download or “stream” from Figma directly.
+
+## Inconstancies & Polyfills
+
+Figma has features that beyond the scope of BabylonJS GUI system. Below is a list of some of the areas which the plug-in has to make a determination on how to interpret things that Figma is asking for that BabylonJS GUI cannot replicate 1:1.
+
+### Handling Images
+
+Images can’t be represented as binary or BASE64 inside the json file so currently the plugin will just set the image src too `source=“./imageControlDefault.jpg”`. After exporting the json its recommended you export your images separately and correct the source addresses either in the JSON file or in code.
```

Comment:
Typos:

*  ``set the image src too`` should be ``set the image src to``?
* ``its recommended`` should be ``it's recommended``?